aioftp/client.py

```diff
@@ -120,7 +120,8 @@ def __init__(self,
                  read_speed_limit=None,
                  write_speed_limit=None,
                  path_timeout=None,
-                 path_io_factory=pathio.PathIO):
+                 path_io_factory=pathio.PathIO,
+                 list_compatibility_level=2):
```

Comment:
The problem is when we need to test LIST fallback, we will need a server that doesn't support MLSD. We will need to add an option to the server anyway to not support MLSD if that is to be tested. The alternative is to use ftp.gnu.org, which may support MLSD in the future, and requires the internet to run test cases. We can simply reset `list_compatibility_level` every time we connect to a new server.